### ISSUE TITLE
JQuery range slider causes bundling failure

### ISSUE THREAD

**Issue by ilkayilknur** [XomniCloud/xomni-cms-vnext#16]

**Comment by daronyondem** [comment]
Did we have a proposed fixed for this? I remember we had plans to rewrite the whole thing in case it takes too much time to fix it. Am I correct? Can you help me remember the latest discussion we had around this particular issue?

**Comment by ilkayilknur** [comment]
@daronyondem yes you're right. We talked the re-implementation of slider. This issue is about rewriting the slider in client analytics summary page.

**Comment by daronyondem** [comment]
Thanks! Will keep this hanging down my backlog for now.
